app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -100,6 +100,8 @@ const validate = createValidator({
 });
 
 const MeetingEditor = () => {
+  const [formKey, setFormKey] = useState(0);
+  const [reportValue, setReportValue] = useState('');
```

Comment:
Could it be possible to extract this value from the `form` variable, or is it just neater with a new useState?